ToolBox/CoinApiDataConverter/CoinApiDataConverter.cs

```diff
@@ -45,15 +45,21 @@ public class CoinApiDataConverter
         private readonly DirectoryInfo _rawDataFolder;
         private readonly DirectoryInfo _destinationFolder;
         private readonly DateTime _processingDate;
+        private readonly string _market;
 
         /// <summary>
         /// CoinAPI data converter.
         /// </summary>
         /// <param name="date">the processing date.</param>
         /// <param name="rawDataFolder">path to the raw data folder.</param>
         /// <param name="destinationFolder">destination of the newly generated files.</param>
-        public CoinApiDataConverter(DateTime date, string rawDataFolder, string destinationFolder)
+        /// <param name="market">The market to process (optional). Defaults to processing all markets in parallel.</param>
+        public CoinApiDataConverter(DateTime date, string rawDataFolder, string destinationFolder, string market = null)
         {
+            _market = string.IsNullOrWhiteSpace(market) 
+                ? null 
+                : market;
```

Comment:
For it to be less error prone we might add `market.ToLower()`, since Lean markets values are lower case